src/routes/mod_creation.rs

```diff
@@ -421,10 +410,30 @@ async fn mod_create_inner(
 
     indexing_queue.add(index_mod);
 
+    let response = crate::models::mods::Mod {
+        id: mod_id,
+        team: crate::models::ids::TeamId(team_id.0 as u64),
+        title: mod_builder.title.clone(),
+        description: mod_builder.description.clone(),
+        body_url: mod_builder.body_url.clone(),
+        published: now,
+        downloads: 0,
+        categories: create_data.categories.clone(),
+        versions: mod_builder
+            .initial_versions
+            .iter()
+            .map(|v| crate::models::ids::VersionId(v.version_id.0 as u64))
```

Comment:
Same as above, `v.version_id.into()` works here. (most of the database Ids implement `Into` for the API model ids.)